packages/app/src/systems/Account/pages/Accounts/Accounts.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, Icon } from '@fuel-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+import { MOCK_ACCOUNTS } from '../../__mocks__';
+import { AccountList } from '../../components';
+import { useAccount } from '../../hooks';
+
+import { Layout, Pages } from '~/systems/Core';
+
+export const Accounts = () => {
+  const navigate = useNavigate();
+  // TODO fix: change to grab multiple accounts
+  const { account, isLoading } = useAccount();
+
+  return (
+    <Layout title="Accounts" isLoading={isLoading}>
+      <Layout.TopBar onBack={() => navigate(Pages.wallet())} />
+
+      <Layout.Content>
+        {account && <AccountList accounts={MOCK_ACCOUNTS} />}
+      </Layout.Content>
+      <Layout.BottomBar>
+        <Button
```

Comment:
The button to add an account can be removed for now until we have the create account page ready. This would avoid users to complain about add account not working.